browser/scripts/application.js

```diff
@@ -1431,7 +1431,12 @@ Application.prototype.onOpenClicked = function() {
 
 
 Application.prototype.onChatDisplayClicked = function() {
-	$('.chat-users').toggle();
+	if (!$('.chat-users').hasClass('collapsed')) {
+		$('.chat-users').toggle();
+	}
+	else {
+		$('.chat-users').removeClass('collapsed').height($('.chat-tabs').height+$('.chat').height)
```

Comment:
just a tip: caching the $() lookups into variables could speed it up (very slightly) and make it more readable